assets/iam-policy.json

```diff
@@ -38,7 +38,8 @@
                 "elasticloadbalancing:DescribeTargetGroups",
                 "elasticloadbalancing:DescribeTargetGroupAttributes",
                 "elasticloadbalancing:DescribeTargetHealth",
-                "elasticloadbalancing:DescribeTags"
+                "elasticloadbalancing:DescribeTags",
+                "elasticloadbalancing:DescribeTrustStores"
```

Comment:
Is updating the iam-policy automated at all, or do you just have to manually look at https://github.com/openshift/aws-load-balancer-controller/blob/d0c13bf1576965a3b65fc09ebce94ed9f86833a2/docs/install/iam_policy.json to see if anything changed and manually sync it?

Reply:
For now, the process is manual. Initially I planned to automate it in this PR and even created a dedicated hack file for that purpose. However, I noticed that the semantic difference can sometimes be much smaller than the byte-by-byte difference because certain statements might be reshuffled upstream.

In this particular case, the semantic change was limited to [adding the elasticloadbalancing:DescribeTrustStores action](https://github.com/openshift/aws-load-balancer-controller/commit/d0c13bf1576965a3b65fc09ebce94ed9f86833a2#diff-18d547fa3761f3fd92e3c94aced34137247a3ebd73b3f547491dffc10859b712) for the mTLS support (which we don't support yet).

I couldn’t find a straightforward way to sort the upstream policy that would minimize the diff while avoiding the risk of losing statements. As a result, I decided to keep the process manual so that multiple people can validate the changes.

Reply:
Makes sense. If someone in the future missed a iam-policy update on a rebase, would you expect it to get caught by E2E tests? Or is it a solid "maybe"?

Reply:
Not as solid as I would like it to be. The e2e tests cover only [the scenarios described in the docs](https://docs.openshift.com/container-platform/4.17/networking/networking_operators/aws_load_balancer_operator/configuring-aws-load-balancer-operator.html). If IAM policy changes go beyond this - we may miss it.